protopipe/pipeline/utils.py

```diff
@@ -325,3 +344,113 @@ def prod3b_array(fileName, site, array):
                 raise ValueError(
                     f"\033[91m ERROR: Array {array} not supported. \033[0m"
                 )
+
+
+def effective_focal_lengths(camera_name):
+    """Provide the effective focal length for the required camera.
+
+    Data comes from Konrad's table.
+
+    Parameters
+    ----------
+    camera_name : str
+        Name of the camera from ctapipe.instrument.CameraDescription
+
+    Returns
+    ----------
+    eff_foc_len : float
+        Effective focal length in meters
+
+    """
+
+    effective_focal_lengths = {
+        "LSTCam": 29.30565 * u.m,
+        "NectarCam": 16.44505 * u.m,
+        "FlashCam": 16.44505 * u.m,
+        "ASTRICam": 2.15191 * u.m,
+        "SCTCam": 5.58310 * u.m,
+        "CHEC": 2.30913 * u.m,
+        "DigiCam": 5.69705 * u.m,
+    }
+
+    eff_foc_len = effective_focal_lengths[camera_name]
+
+    return eff_foc_len
+
+
+def camera_radius(camid_to_efl, cam_id="all"):
+    """Get camera radii.
+
+    Inspired from pywi-cta CTAMarsCriteria, CTA Mars like preselection cuts.
+    This should be replaced by a function in ctapipe getting the radius either
+    from  the pixel poisitions or from an external database
+
+    Note
+    ----
+    average_camera_radius_meters = math.tan(math.radians(average_camera_radius_degree)) * foclen
+    The average camera radius values are, in degrees :
+    - LST: 2.31
+    - Nectar: 4.05
+    - Flash: 3.95
+    - SST-1M: 4.56
+    - GCT-CHEC-S: 3.93
+    - ASTRI: 4.67
+
+    """
+
+    average_camera_radii_deg = {
+        "ASTRICam": 4.67,
+        "CHEC": 3.93,
+        "DigiCam": 4.56,
+        "FlashCam": 3.95,
+        "NectarCam": 4.05,
+        "LSTCam": 2.31,
+        "SCTCam": 4.0,  # dummy value
+    }
+
+    if cam_id in camid_to_efl.keys():
+        foclen_meters = camid_to_efl[cam_id]
+        average_camera_radius_meters = (
+            math.tan(math.radians(average_camera_radii_deg[cam_id])) * foclen_meters
+        )
+    elif cam_id == "all":
+        print("Available camera radii in meters:")
+        for cam_id in camid_to_efl.keys():
+            print(f"* {cam_id} : {camera_radius(camid_to_efl, cam_id)}")
+        average_camera_radius_meters = 0
+    else:
+        raise ValueError("Unknown camid", cam_id)
+
+    return average_camera_radius_meters
+
+
+def CTAMARS_radii(camera_name):
+
```

Comment:
This probably doesn't even need to be a function, right? Nothing inside is computed or changes.  Why not just make it a module-level variable called CTAMARS_RADII?

Reply:
Yes, you're right.
Would you prefer I do it here or I keep it a small fix for later?